gemrb/core/GameData.cpp

```diff
@@ -198,13 +198,13 @@ int GameData::GetTableIndex(const ResRef &resRef) const
 	return -1;
 }
 /** Gets a Loaded Table by its index, returns NULL on error */
-Holder<TableMgr> GameData::GetTable(size_t index) const
+std::shared_ptr<TableMgr> GameData::GetTable(size_t index) const
 {
 	if (index >= tables.size()) {
-		return NULL;
+		return {};
```

Comment:
Is this just a shorthand for nullptr or does it actually construct an object? We null-check most of these calls later.

Reply:
It constructs an object without a managed pointer. But it has an [overloaded bool](https://en.cppreference.com/w/cpp/memory/shared_ptr/operator_bool) so we can leave these checks as-is, since this is false then.

Reply:
`return {};` and `return nullptr` are equivalent, but `NULL` is an integer type.